subprojects/docs/src/docs/release/notes.md

```diff
@@ -61,6 +61,72 @@ With this release of Gradle, the console displays any test worker processes exec
 Tasks of types [ScalaCompile](dsl/org.gradle.api.tasks.scala.ScalaCompile.html) and [ScalaDoc](dsl/org.gradle.api.tasks.scala.ScalaDoc.html) provided by the [scala](userguide/scala_plugin.html) plugin are now cacheable.
 This means they will make use of the build cache when activated.
 
+### New API for safe, fast concurrent work execution
+
+Gradle 4.1 introduces the [Worker API](userguide/custom_tasks.html#worker_api), a new API for safe, parallel, and asynchronous execution of build logic.  This API allows for:
```

Comment:
"asynchronous execution of build _logic_" sounds like parallel configuration vs parallel execution of work.